sklearn/datasets/tests/test_common.py

```diff
@@ -0,0 +1,10 @@
+"""Test loaders for common functionality.
+"""
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_true
+
+
+def check_return_X_y(bunch, X_y_tuple):
+    assert_true(isinstance(X_y_tuple, tuple))
+    assert_array_equal(X_y_tuple[0].shape, bunch.data.shape)
```

Comment:
Shapes should not be arrays. Just use assert ==

Reply:
Ok, switched in https://github.com/scikit-learn/scikit-learn/pull/10774/commits/5f73583c99c773fbc204b63afe2acce793fe58fa